packages/api/src/hooks/useAuthorize.ts

```diff
@@ -12,7 +12,7 @@ const useAuthorize = () => {
 
     const { data, ...rest } = useQuery('authorize', {
         payload: { authorize: current_token || '' },
-        options: { enabled: Boolean(current_token) },
+        options: { enabled: Boolean(current_token), keepPreviousData: true },
```

Comment:
does this have effects only on wallets, or is it used elsewhere too?

Reply:
@wojciech-deriv as I know we are using `useQuery` just for `wallets` package now. Anyway, I'm pretty sure this `keepPreviousData` can't break anything, it's just for UI

<img width="893" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/08b91531-aa33-46a7-a01f-71ccb579f3e8">

Reply:
could it cause a component to use stale data from previous authorisation e.g. while submitting a transfer, or reseting a balance, in case of new authorisation data being delayed/erorrer (e.g. poor network connection, which is common among our customers). E.g. lets say that navigation worked correctly and moved on to new route/component, but authorisation is still providing stale data from previous account. 

I think best way to go would be to somehow explicitly support loading and error state - at least it ensures no weird edge cases.

Reply:
Had a thought to add this before, and farzin was saying better not to hide the actual problem. I think multiple authorization could solve this issue https://app.clickup.com/t/20696747/PLA-623

Reply:
@wojciech-deriv Understood your point, didn't think about this before. I'm not sure it could be a problem in our project, but we don't need to risk. Anyway, we can add this property to the hook which works with account_list endpoint when it will be created by BE team, but for now maybe would be better leave it as it now.

@farhan-nurzi-deriv yeah, hope this issue will be solved by this solution, thank you!